src/VisualStudio/Core/Def/ProjectSystem/OpenTextBufferProvider.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.ErrorReporting;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    /// <summary>
+    /// A class that provides access to the currently open list of files in Visual Studio. You are able to ask for the text buffer for a document on any thread; events are raised
+    /// on thread, and any method that provides a <see cref="IVsHierarchy"/> must be used on the UI thread.
+    /// </summary>
```

Comment:
I assume this is what you meant
```suggestion
    /// A class that provides access to the currently open list of files in Visual Studio. You are able to ask for the text buffer for a document on any thread; events are raised
    /// on the UI thread, and any method that provides a <see cref="IVsHierarchy"/> must be used on the UI thread.
    /// </summary>
```

Reply:
Yes I looks like I deleted a few